bucket/PowerSession.json

```
@@ -1,23 +1,23 @@
 {
-    "version": "1.4.0",
+    "version": "1.4.2",
     "description": "Powershell session recorder",
     "homepage": "https://github.com/ibigbug/PowerSession",
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/ibigbug/PowerSession/releases/download/release-v1.4.0/PowerSession.exe",
+            "url": "https://github.com/ibigbug/PowerSession/releases/download/v1.4.2/PowerSession-win10-x64.exe",
             "hash": "d524cb145abe75743775e3bf1672608cea2214adbdd8c872b33744e126145009"
         }
     },
     "bin": "PowerSession.exe",
     "checkver": {
         "github": "https://github.com/ibigbug/PowerSession",
-        "regex": "tag/release-v([\\d.]+)"
+        "regex": "tag/v([\\d.]+)"
     },
```

There should be no need for regex since you are using GitHub releases

```suggestion
    },
```